pkg/webui/console/containers/gateway-managed-gateway/connection-settings/connections/index.js

```diff
@@ -0,0 +1,173 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import classnames from 'classnames'
+
+import gatewayIcon from '@assets/misc/gateway.svg'
+
+import Icon from '@ttn-lw/components/icon'
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import {
+  connectionMessageMap,
+  exampleConnectionsResponse,
+  getCellularDetails,
+  getConnectionType,
+  getDetails,
+  getEthernetDetails,
+  getWifiDetails,
+  isConnected,
+  NETWORK_INTERFACE_TYPES,
+} from '@console/containers/gateway-managed-gateway/connection-settings/connections/utils'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import m from './messages'
+
+import style from './connections.styl'
+
+const ManagedGatewayConnections = () => {
+  const managedGateway = exampleConnectionsResponse
+
+  const gatewayControllerConnection =
+    managedGateway.result.controller_connection.network_interface_type
+  const gatewayServerConnection =
+    managedGateway.result.gateway_server_connection.network_interface_type
+
+  const controllerConnectionIsSpecified =
+    gatewayControllerConnection !== NETWORK_INTERFACE_TYPES.UNSPECIFIED
+  const serverConnectionIsSpecified =
+    gatewayServerConnection !== NETWORK_INTERFACE_TYPES.UNSPECIFIED
+
+  const isCellularConnected = isConnected(
+    managedGateway.result.cellular_backhaul.network_interface.status,
+  )
+
+  const isWifiConnected = isConnected(managedGateway.result.wifi_backhaul.network_interface.status)
+
+  const isEthernetConnected = isConnected(
+    managedGateway.result.ethernet_backhaul.network_interface.status,
+  )
+
+  const getIsConnectedDiv = isConnected => (
+    <div className="d-flex al-center gap-cs-xxs">
+      <Icon icon={isConnected ? 'valid' : 'cancel'} />
+      <Message content={isConnected ? sharedMessages.connected : sharedMessages.disconnected} />
+    </div>
+  )
+
+  return (
+    <div className={style.root}>
+      <Message className="fw-bold mt-0" component="h2" content={sharedMessages.managedGateway} />
+      <div className={classnames(style.top, 'd-flex j-between al-center pb-cs-xs mb-cs-xs')}>
+        <div>
+          <p className="m-0">
+            <Message content={sharedMessages.hardwareVersion} />:{' '}
+            {managedGateway.result.entity.version_ids.hardware_version}
+          </p>
+          <p className="m-0">
+            <Message content={sharedMessages.firmwareVersion} />:{' '}
+            {managedGateway.result.entity.version_ids.firmware_version}
+          </p>
+          <div className="d-flex al-center gap-cs-xxs mb-cs-s mt-cs-xxs">
+            <Icon icon="cloud" />
+            {managedGateway.result.system_metrics.temperature} &deg;C
+          </div>
+          <Link.Anchor primary href="/gateways/adding-gateways">
+            {m.officialDocumentation.defaultMessage}
```

Comment:
This should then be a `<Message />`
```suggestion
            <Message content={m.officialDocumentation} />
```